src/manifold/src/impl.cpp

```diff
@@ -902,4 +903,59 @@ SparseIndices Manifold::Impl::VertexCollisionsZ(
   else
     return collider_.Collisions<false, false>(vertsIn);
 }
+
+struct GetDistanceTriangleTriangleSquared {
```

Comment:
These private functors actually need to live up above in the anonymous namespace to avoid potential name collisions. However, we've actually been transitioning the code base away from these functors to lambdas, especially for simple functions like this. The functors were mainly because CUDA support of lambdas was iffy, but now that we've moved off of CUDA, lambdas are generally easier to read.

Reply:
Done!